src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
 body {
   margin: 0;
+  width: 1024px;
```

Comment:
Setting a fixed width for the body can cause layout issues on different screen sizes. Consider using a responsive approach, such as percentages or media queries, to ensure the layout adapts to various devices.